modules/ROOT/pages/index.adoc

```diff
@@ -18,14 +18,14 @@ security, and scalability.
 
 Anypoint Studio and Flow Designer support Mule application development.
 
-== Domains
+== Mule Domains
 
 Domains enable you to share global configurations that Mule applications need to
 reuse, such as default error handlers, shared properties, scheduler pools, and
-component configurations.
+component configurations. You can only deploy domains when running Mule runtime engine on-premises.
```

Comment:
"on-premises" : You don't need to use the hyphen because "on-premises" is not being used as an adjective.